tests/test_dataset.py

```diff
@@ -24,7 +25,12 @@ def test_compute_functions(dataset):
     for calc in types:
         result = dataset.compute(calc)
         assert isinstance(result, dict)
+        # Also test output for all types
+        tmpfile = tempfile.NamedTemporaryFile()
+        result.write_results(tmpfile)
```

Comment:
should this be 
`dataset.write_results(tmpfile)`

Reply:
Mmh, sorry, I thought I did test before sending but ran the file through python instead of pytest. 

I think I misuse the NamedTemporaryFile() here, let me have a look on my side...

Reply:
got it : line 30 should be 
```dataset.write_results(tmpfile.name)```